source/MissionTestMod/MissionTestMod.csproj.user

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <StartArguments>/singleplayer /client _MODULES_%2aNative%2aSandBoxCore%2aSandBox%2aStoryMode%2aMissionTestMod%2a_MODULES_</StartArguments>
+    <StartAction>Program</StartAction>
+    <StartProgram>C:\Users\allen\source\repos\BannerlordCoop\mb2\bin\Win64_Shipping_Client\Bannerlord.exe</StartProgram>
```

Comment:
Convert this to relative paths or remove file from merge

Reply:
good eye! will update